tests/unit/i18n.ts

```diff
@@ -22,16 +22,15 @@ import bundle from '../support/mocks/common/main';
 import partyBundle from '../support/mocks/common/party';
 
 registerSuite('i18n', {
-
 	before() {
 		// Load the CLDR data for the locales used in the tests ('en' and 'fr');
-		return fetchCldrData([ 'en', 'fr' ]).then(() => {
+		return fetchCldrData(['en', 'fr']).then(() => {
 			switchLocale('en');
 		});
 	},
 
 	afterEach() {
-		const loadCldrData = <any> cldrLoad.default;
+		const loadCldrData = <any>cldrLoad.default;
```

Comment:
I'm surprised the lack of space after `<any>` doesn't cause problems when running `grunt`. Elsewhere we've had to change `<any> value` to `(value as any)`.